### ISSUE TITLE
Any update on AngularJS support as in documentation?

### ISSUE THREAD

**Issue by afidegnum** [dpgaspar/Flask-AppBuilder#482]
HI, I m in dire need to implement an Ajax framework, 

I was planning the ELM implementation till i read you have AngularJS in perspective. 

What's the status?

**Comment by dpgaspar** [comment]
Hi @afidegnum,

AngularJS implementation would be great i agree, i have tried a small prototype, but this would probably break a lot on F.A.B. 

The status is stopped i don't have spare time to do it.

**Comment by AustinJim** [comment]
Be sure to look at the examples, especially quickhowto2 for examples of AngularJS usage with FAB.

**Comment by afidegnum** [comment]
i have tried quite few Ajax frameworks too and i have centered into ELM, but the issue everything need to be prepared from start, Vuejs seems to be good too but the clusters of dependencies add some complication instead of fast development. AngularJS too appears tedious to steer along the way.

**Comment by AustinJim** [comment]
I have used AngularJS at work and started a personal project with it. But, I find it just too complicated and the up front investment in time for learning and set up is large. I also looked at Knockout.js which is much simpler than AngularJS but dose provide two way finding. The problem I had with Knockout is that it doesn't play well with Bootstrap or Jquery Mobile.
Finally I decided to roll my own using jquery's ajax capabilities. Now my project is progressing much faster.
Good luck!
